### PR TITLE
readmes: more updates

### PR DESCRIPTION
**Pull request by divyenduz**
related https://github.com/prisma/prisma2-e2e-tests/issues/52

### CODE REVIEW

File: platforms/netlify-ci-build/README.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
 # Introduction
 
-Prisma and Netlify (beta build) integration.
+Prisma and Netlify [beta build](https://build-beta.netlify.com/) integration.
 
 # Friction Points
 
 - https://github.com/netlify/build/issues/730
+- Not zero-config, binary target configuration needed
```
Comment (janpio) [inline platforms/netlify-ci-build/README.md]: Isn't this the same thing as the issue above? If so, these should be combined into 1 friction point I guess?

File: platforms/netlify/README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Introduction
 
-Prisma and Netlify (current build bot) integration.
+Prisma and Netlify (current build CI) integration.
 
 # Friction Points
 
-- Since Netlify's current build bot doesn't use the newer version of `zip-it-and-ship-it`, we need to bundle the binaries ourselves. See `build.sh`.
+- Since Netlify's current build CI doesn't use the newer version of `zip-it-and-ship-it`, we need to bundle the binaries ourselves. See `build.sh`.
```
Comment (janpio) [inline platforms/netlify/README.md]: `zip-it-and-ship-it` could be linked to the repo

File: platforms/netlify-ci-build/README.md (hunk 1 above)
Comment (divyenduz) [reply platforms/netlify-ci-build/README.md]: :(

File: platforms/netlify/README.md (hunk 2 above)
Comment (divyenduz) [reply platforms/netlify/README.md]: For future!

File: platforms/netlify-ci-build/README.md (hunk 1 above)
Comment (janpio) [reply platforms/netlify-ci-build/README.md]: Yes? No?

Comment (divyenduz) [reply platforms/netlify-ci-build/README.md]: :( is always a yes!

**Pull request merged**